components/Msig/CreateConfirm.test.tsx

```diff
@@ -0,0 +1,113 @@
+import '@testing-library/jest-dom/extend-expect'
+import {
+  act,
+  cleanup,
+  getByRole,
+  getByText,
+  render,
+  RenderResult
+} from '@testing-library/react'
+import { useMessageQuery } from '@glif/react-components'
+
+import composeMockAppTree from '../../test-utils/composeMockAppTree'
+import { CreateConfirm } from './CreateConfirm'
+import { PAGE } from '../../constants'
+import { flushPromises } from '../../test-utils'
+
+jest.mock('../../MsigProvider')
+jest.mock('@glif/react-components', () => {
+  const original = jest.requireActual('@glif/react-components')
+  return {
+    ...original,
+    useMessageQuery: jest.fn(({ variables }) => ({
+      data: { message: { cid: variables.cid } }
+    }))
+  }
+})
```

Comment:
@Schwartz10  I'm wondering if this is the right approach for mocking useMessageQuery here

Reply:
https://github.com/glifio/safe/blob/primary/test-utils/composeMockAppTree/index.jsx#L11

Check out this `MockedProvider` through Apollo - i think this is considered the "right" way

Reply:
Reopening in a new ticket here https://github.com/glifio/safe/issues/108